### PR TITLE
Trading component

### PR DESCRIPTION
**Pull request by vinu-deriv**

### CODE REVIEW

**Review by aswathy-deriv — APPROVED** [review]

File: src/App.module.scss [hunk 1]
```diff
@@ -123,8 +123,9 @@ a {
 }
 
 .content {
-  height: calc(100% - 80px);
+  height: calc(100% - 81px);
```
Comment (farrah-deriv) [inline src/App.module.scss:126]:
```suggestion
  height: calc(100% - 80px);
```

File: src/constants/trade-config.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  //     trade_types: ['MULTUP', 'MULTDOWN'],
+  //     basis: ['stake'],
+  //     components: [
+  //         'take_profit',
+  //         'stop_loss',
+  //         ...(shouldShowCancellation(symbol) ? ['cancellation'] : []),
+  //         ...(shouldShowExpiration(symbol) ? ['expiration'] : []),
+  //     ],
+  //     config: { hide_duration: true },
+  // },
```
Comment (farrah-deriv) [inline src/constants/trade-config.js]: can we remove this? or if this is necessary, please add TODO so we can work on this later.

File: src/routes/reports/reports.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const Reports = () => {
+  return <div>Will be implemented soon</div>;
```
Comment (farrah-deriv) [inline src/routes/reports/reports.tsx]: please add TODO

**Review by likhith-deriv — APPROVED** [review]

File: src/routes/trade/options-trade.jsx [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+          />
+          <p>USD</p>
+        </div>
+
+        <div class={`${styles["button"]} ${styles["buy-sell"]}`}>
+          <div class={styles["buy-sell__buy-wrapper"]}>
+            {buySellButtonWrapper(proposal_buy)}
+            <button
+              class={`${styles["buy-sell__buy"]} ${
+                !proposal_buy.id ? styles["buy-sell__buy--disabled"] : ""
```
Comment (farrah-deriv) [inline src/routes/trade/options-trade.jsx]: can we use `classNames` instead

File: src/routes/trade/options-trade.jsx [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+              }}
+            >
+              {trade_types.title.split("/")[0]}
+            </button>
+          </div>
+          <div class={styles["buy-sell__sell-wrapper"]}>
+            {buySellButtonWrapper(proposal_sell)}
+            <button
+              class={`${styles["buy-sell__sell"]} ${
+                !proposal_sell.id ? styles["buy-sell__sell--disabled"] : ""
```
Comment (farrah-deriv) [inline src/routes/trade/options-trade.jsx]: same here please use `classNames`

File: src/App.module.scss (hunk 1 above)
Comment (vinu-deriv) [reply src/App.module.scss:126]: will be taken care by @likhith-deriv or me in later Pr

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**